src/custom_source_buffers/text/html/html_text_source_buffer.ts

```diff
@@ -171,7 +171,7 @@ export default class HTMLTextSourceBuffer
         // to spread the time error, we divide the regular chosen interval.
         // As the clock is also based on real video events, we cannot just
         // divide by two the regular interval.
-        const time = Math.max(this._videoElement.currentTime -
+        const time = Math.max(this._videoElement.currentTime +
                               MAXIMUM_HTML_TEXT_TRACK_UPDATE_INTERVAL / 2000,
```

Comment:
Why divided by 2000 ?

Reply:
Ok. Maybe it may be clearer.
If you decide to check for the current time, we may have issues if the cues for current time and current time + interval are not the same. We still should display the cue for current time + interval.

Reply:
Done